<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>background</class>
 <widget class="QDialog" name="background">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>591</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1060</width>
    <height>591</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1060</width>
    <height>591</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color:rgb(54, 54, 54)

</string>
  </property>
  <widget class="QPushButton" name="disconnect_button">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>250</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>51</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(184, 184, 184);
font: 8pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>510</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>51</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(184, 184, 184);
font: 8pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="message_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>891</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>891</width>
     <height>51</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>891</width>
     <height>51</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(184, 184, 184);
color: rgb(243,243,243);
border-radius:20px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>  Type message here...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditor">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>891</width>
     <height>451</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>891</width>
     <height>451</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>891</width>
     <height>451</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>184</red>
         <green>184</green>
         <blue>184</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(184, 184, 184);
border-radius:20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Chat/QT5 chat stuff/background.qrc"/>
  <include location="Chat/QT5 chat stuff/bakgrund.qrc"/>
 </resources>
 <connections/>
</ui>
